marketplace/release-notes/9.1.1.txt

```diff
@@ -0,0 +1 @@
+We now use same context to create logs to make sure that the object and the log are committed in the same transaction
```

Comment:
This is a behavior change, so we should be more explicit: failure to commit an object also means failure to commit its log line.